### PR TITLE
Global: Fix some typos

### PR DESCRIPTION
**Pull request by mishmetall**
Most are typos, but some words changed to en-GB version.

### CODE REVIEW

File: libraries/AC_Fence/AC_Fence.cpp [hunk 1]
```diff
@@ -494,7 +494,7 @@ bool AC_Fence::check_fence_polygon()
 /// check_fence_circle - returns true if the circle fence (defined via
 /// parameters) has been freshly breached.  May also set up a backup
 /// fence outside the fence and return a fresh breach if that backup
-/// fence is breaced.
+/// fence is braced.
```
Comment (Hwurzburg) [inline libraries/AC_Fence/AC_Fence.cpp:497]: breached

File: libraries/AP_DDS/README.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Run the simulator with the following command. If using UDP, the only parameter y
 | SERIAL1_PROTOCOL | Set this to 45 to use DDS on the serial port | 0 |
 ```console
 # Wipe params till you see "AP: ArduPilot Ready"
-# Select your favorite vehicle type
+# Select your favourite vehicle type
```
Comment (Hwurzburg) [inline libraries/AP_DDS/README.md:114]: was not misspelled

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
nice cleanup

File: libraries/AP_DDS/README.md (hunk 2 above)
Comment (peterbarker) [reply libraries/AP_DDS/README.md:114]: > was not misspelled

Well... that's a matter of some debate :-)  I think the key thing is that we don't correct one way or the other and accept either.

**Review by peterbarker — CHANGES_REQUESTED** [review]
commit message needs fixing.  I think this is actually a reasonable one to go in with a `global: ` prefix, something we don't do very often.

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
squash to single commit and I will merge

**Review by Hwurzburg — APPROVED** [review]
sorry meant I will approve for merge..marking for next devcall

**Comment by rmackay9** [comment]
Yes, nice to have proper spelling everywhere, it shows we care about the details.

Don't we have a script to break the commit up into a single commit for each vehicle or library?  I'd actually prefer that.  I worry this will draw one of those lines in the code which makes backports difficult.

**Comment by Hwurzburg** [comment]
Tools/gittools/git-subsystems-split will do it...you right this is code, not wiki, need the lib splits

**Review by tridge — APPROVED** [review]

**Comment by peterbarker** [comment]
Thanks @mishmetall , we fixed up one word which was incorrect (consistent vs constant) and I split it up into separate commits (as people decided on DevCall).

Marked for merging after CI has passed.

**Pull request merged**